src/CoreBundle/Controller/SecurityController.php

```diff
@@ -41,6 +45,18 @@ public function loginJson(AuthenticationUtils $authenticationUtils): Response
         //$error = $authenticationUtils->getLastAuthenticationError();
         //$lastUsername = $authenticationUtils->getLastUsername();
 
+        // Log of connection attempts
+        $params = json_decode($request->getContent(), true);
+        $trackELoginRecord = new TrackELoginRecord();
+        $trackELoginRecord
+            ->setUsername($params['username'])
```

Comment:
No es necesario obtener el username del contenido de la request, porque más abajo se obtiene el usuario que inició sesión satisfactoriamente ($user = $this->getUser()).